docs/sources/trackers/javascript-trackers/web-tracker/cross-domain-tracking/index.md

```diff
@@ -4,23 +4,351 @@ date: "2022-08-30"
 import TabItem from '@theme/TabItem';
-import CrossDomain from "@site/docs/reusable/javascript-tracker-cross-domain/_index.md"
 ```
 
+The JavaScript tracker can add an additional parameter named `_sp` to the querystring of outbound links. This process is called "link decoration".
+
+Configure cross-domain tracking using the `useExtendedCrossDomainLinker` or `crossDomainLinker` fields of the [configuration object](/docs/sources/trackers/javascript-trackers/web-tracker/tracker-setup/initialization-options/index.md). Choose between extended or short format for the `_sp` querystring parameter:
+
+* Short format: choose which links to decorate with the core parameters only
+* Extended format: the querystring can include more parameters, but the tracker will decorate all cross-domain links
+
+If you set both configuration options, the tracker will use the extended format.
+
+You can include the following properties in the `_sp` querystring:
+
+| Property        | Description                                    | Extended | Short |
+| --------------- | ---------------------------------------------- | -------- | ----- |
+| `domainUserId`  | Current tracker-generated UUID user identifier | ✅        | ✅     |
+| `timestamp`     | Current epoch timestamp, ms precision          | ✅        | ✅     |
+| `sessionId`     | Current session UUID identifier                | ✅        |       |
+| `subjectUserId` | Custom business user identifier                | ✅        |       |
+| `sourceId`      | Application identifier                         | ✅        |       |
+| `platform`      | Platform of the current device                 | ✅        |       |
+| `reason`        | Link text or extra information                 | ✅        |       |
+
+For example,
+
+```
+appSchema://path/to/page?_sp=domainUserId.timestamp.sessionId.subjectUserId.sourceId.platform.reason
+```
+
+Link decoration makes these values visible in the `url` field of events sent by the JavaScript tracker on the destination page. The enrichment process will use them to populate the `refr_domain_userid` and `refr_dvce_tstamp` fields for all events fired on the destination page, where the URL includes the `_sp` parameter.
+
+The querystring is added only to the events on the destination page: it doesn't persist throughout the user's session.
+
+:::tip
+
+If you enable link decoration, aim for at least one event to be tracked on the page. The tracker writes the `domain_userid` to a cookie when it tracks an event. If the cookie doesn't exist when the user leaves the page, the tracker will generate a new ID for them when they return, rather than keeping the old ID. This can make user stitching difficult.
+
+:::
+
+## Extended format
+
+The extended `_sp` value always includes `domainUserId` and `timestamp`. The other properties are configurable using the `ExtendedCrossDomainLinkerAttributes` object.
+
+By default, `reason` will be the link text from a cross-domain link.
```

Comment:
This is only the default if you specify `true` for `reason` instead of providing your own callback, the default is that it's not collected at all, only `sessionId` and `sourceId` are added with `useExtendedCrossDomainLinker: true`.